hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/utils/CRLCodec.java

```diff
@@ -78,7 +78,7 @@ public CRLCodec(SecurityConfig securityConfig) {
    * Returns a X509 CRL from the CRL Holder.
    *
    * @param holder - Holder
-   * @return X509Certificate.
+   * @return X509CRL - X509 CRL.
    * @throws CRLException - on Error.
    */
   public static X509CRL get509CRL(X509CRLHolder holder)
```

Comment:
NIT: should we rename the method from get509CRL to getX509CRL?

Reply:
Good point. Done.